Rakefile

```diff
@@ -50,4 +52,4 @@ end
 # Default
 task default: ['style', 'spec', 'integration:vagrant']
 task ec2: ['style', 'spec', 'integration:ec2']
-task test: ['style', 'spec']
+task test: %w(style spec)
```

Comment:
Was there a reason to change this and make the format inconsistent from the other tasks?